src/app/(nextjs_migration)/components/client/button.module.scss

```diff
@@ -0,0 +1,30 @@
+@import "../../../tokens";
+
+.button {
+  font: $text-body-md;
+  font-weight: 500;
+  border: 0;
+  padding: $spacing-sm $spacing-md;
+    border-radius: $border-radius-sm;
+  cursor: pointer;
+  display: inline-block;
+  line-height: 1;
+
+  &.primary {
+    color: white;
+    background-color: $color-blue-50;
+  }
+
+  &.secondary {
+    color: $color-blue-50;
+      border: solid 2px $color-blue-50;
+      background-color: transparent;
+    box-shadow: rgba(0, 0, 0, 0.15) 0px 0px 0px 1px inset;
```

Comment:
Same here (and in `&.large` below) re: weird indentation.
Not sure we're using prettier yet or if we need to check to see if it's watching .scss files.